README.md

```diff
@@ -305,13 +305,23 @@ There are some settings that can be configured via a `settings.json` file:
 - VS Code: These settings will show up in the settings window
 - LanguageClient-neovim: Create this file in `$projectdir/.vim/settings.json` or set `g:LanguageClient_settingsPath`
 
-## Explicit Configuration
+## Project Configuration
 
-**For a full explanation of possible configuration, we refer to [hie-bios/README](https://github.com/mpickering/hie-bios/blob/master/README.md).**
+**For a full explanation of possible configurations, refer to [hie-bios/README](https://github.com/mpickering/hie-bios/blob/master/README.md).**
 
-The user can place a `hie.yaml` file in the root of the workspace which
-describes how to setup the environment. For example, to explicitly state
-that you want to use `stack` then the configuration file would look like:
+HIE will attempt to automatically detect your project configuration and set up
+the environment for GHC. 
+
+| `cabal.project` | `stack.yaml` | `*.cabal` | Project selected |
+|-----------------|--------------|-----------|------------------|
+| ✅              | -            | -         | Cabal v2         |
+| ❌              | ✅           | -         | Stack            |
+| ❌              | ❌           | ✅        | Cabal (v2 or v1) |
+| ❌              | ❌           | ❌        | Plain GHC        |
```

Comment:
Currently, I think there is no support for plain ghc projects.

Reply:
Is this not the direct cradle?

Reply:
Afaik, you have to specify the direct cradle yourself via hie.yaml.

Reply:
We should maybe document this potential use-case more, though.